designsafe/middleware.py

```diff
@@ -19,13 +19,13 @@
 
 class DesignSafeSupportedBrowserMiddleware:
     """
-    Middleware to check if the user is running Chrome or Firefox and 
+    Middleware to check if the user is running Chrome or Firefox and
     flash a warning otherwise.
     """
 
     def process_request(self, request):
         user_agent = request.META['HTTP_USER_AGENT']
-        agent_is_supported = ('Chrome' in user_agent) or ('Firefox' in user_agent)
+        agent_is_supported = ('Chrome' in user_agent) or ('Firefox' in user_agent) or ('FxiOS' in user_agent) or ('CriOS' in user_agent)
```

Comment:
Now that we have 4 user agent strings, referenced both here and in `base.html`, perhaps we should pull them out into a setting, `SUPPORTED_USER_AGENTS = ['Chrome', 'Firefox', 'FxiOS', 'CriOS'], and this would instead become something like
```suggestion
        agent_is_supported = any(agent in user_agent for agent in settings.SUPPORTED_USER_AGENTS)
```

Reply:
That sounds good!

Reply:
I'll make changes to settings.

Reply:
@rstijerina Would there be a simple way to use Django settings in the template as well?

Reply:
@duckonomy Ah yes, we may want to refactor this a bit. You can reference variables in a django template via `context`, and the [messages](https://docs.djangoproject.com/en/1.11/ref/contrib/messages/) framework has an accompanying context processor in `designsafe/context_processors.py`.
1. Add a context variable to the `messages` function of `context_processors` that  evaluates the `agent_is_supported` logic from `middleware.py` (after looking closer I'm not even sure this is active)
2. [Reference the boolean in the `messages.html` template ](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#if)
3. Move the html in `base.html` that displays the warning into the conditional in `messages.html`

Reply:
@rstijerina Thanks for the insight!